stanford_sites_helper.install

```diff
@@ -8,6 +8,29 @@
  */
 function stanford_sites_helper_install() {}
 
+/**
+ * Implements hook_requirements().
+ */
+function stanford_sites_helper_requirements($phase) {
+  $requirements = array();
+
+  switch ($phase) {
+    case 'runtime':
+      $t = get_t();
+      $file_public_path = variable_get('file_public_path', '');
+      $file_private_path = variable_get('file_private_path', '');
+
+      $requirements['stanford_sites_helper'] = array(
+        'title' => $t('File Paths'),
+        'value' => $t('Your public file path is %file_public_path. Your private file path is %file_private_path.', array('%file_public_path' => $file_public_path, '%file_private_path' => $file_private_path)),
+        'severity' => REQUIREMENT_INFO,
```

Comment:
`REQUIREMENT_OK` or `REQUIREMENT_INFO`?

Reply:
INFO as you are not really doing a check for anything but rather, are just displaying information even if it is incorrect.

Reply:
Yeah, I was debating to check whether the path is set but that way be dragons.